### PR TITLE
PR: Deploy to Fly.io issue #52

### PR DESCRIPTION
**Pull request by nelsonic**
This PR creates the needed files to deploy to Fly.io #52

### CODE REVIEW

**Comment by nelsonic** [comment]
```sh
Finished in 0.1 seconds (0.06s async, 0.1s sync)
25 tests, 0 failures

Randomized with seed 781113
----------------
COV    FILE                                        LINES RELEVANT   MISSED
  0.0% lib/app.ex                                      9        0        0
  0.0% lib/app/application.ex                         38        4        4
  0.0% lib/app/mailer.ex                               3        0        0
  0.0% lib/app/release.ex                             28        7        7
  0.0% lib/app/repo.ex                                 5        0        0
100.0% lib/app/todo.ex                               111        7        0
100.0% lib/app/todo/item.ex                           19        2        0
100.0% lib/app_web.ex                                114        2        0
 20.9% lib/app_web/components/core_components.e      621      153      121
  0.0% lib/app_web/components/layouts.ex               5        0        0
100.0% lib/app_web/controllers/error_html.ex          19        1        0
100.0% lib/app_web/controllers/error_json.ex          15        1        0
 68.8% lib/app_web/controllers/item_controller.       98       32       10
 92.3% lib/app_web/controllers/item_html.ex           41       13        1
  0.0% lib/app_web/endpoint.ex                        51        0        0
  0.0% lib/app_web/gettext.ex                         24        0        0
 50.0% lib/app_web/router.ex                          48        6        3
  0.0% lib/app_web/telemetry.ex                       92        5        5
100.0% test/support/conn_case.ex                      38        2        0
 57.1% test/support/data_case.ex                      58        7        3
100.0% test/support/fixtures/todo_fixtures.ex         22        2        0
[TOTAL]  36.9%
----------------
Generating report...

# 😢 
```

**Comment by nelsonic** [comment]
```sh
warning: this clause cannot match because a previous clause at line 1 always matches
  lib/app_web/router.ex:1
```
This is such an unhelpful warning. 🤷‍♂️ (🤦‍♂️)

**Comment by nelsonic** [comment]
Useless warning message is a _known_ issue from _way_ back when: https://github.com/elixir-lang/elixir/issues/7224#issuecomment-1344613407

File: lib/app_web/controllers/item_controller.ex [hunk 1]
```diff
@@ -7,18 +7,21 @@ defmodule AppWeb.ItemController do
   alias App.Repo
 
   def index(conn, params) do
-    item = if not is_nil(params) and Map.has_key?(params, "id") do
-      Todo.get_item!(params["id"])
-    else
-      %Item{}
-    end
+    item =
```
Comment (nelsonic) [inline lib/app_web/controllers/item_controller.ex:10]: all the code changes are from running `mix format`

**Comment by nelsonic** [comment]
@LuchoTurtle I should have spotted that https://github.com/dwyl/phoenix-todo-list-tutorial/pull/55/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485 removed `.travis.yml` but didn't add the corresponding `.github/workflows/ci.yml` ... My bad for not paying enough attention when reviewing.

Please review this and merge when you can. Not urgent. Monday is fine. 👍

**Comment by nelsonic** [comment]
I would obviously _prefer_ the coverage to be **`100%`** but that is not the _focus_ of `this` PR. 💭

**Comment by LuchoTurtle** [comment]
@nelsonic I can work on getting the coverage to 100%. I just need you to merge the other branches (including this one).

**Review by LuchoTurtle — APPROVED** [review]
lgtm 👍

**Pull request merged**